src/util.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Can we name this file something more specific? "util.js" is too broad. I'd be ok if we just named it "object.js".